components/support/webextensions/src/main/java/mozilla/components/support/webextensions/WebExtensionSupport.kt

```diff
@@ -97,6 +98,8 @@ object WebExtensionSupport {
      * @param onUpdatePermissionRequest (optional) Invoked when a web extension has changed its
      * permissions while trying to update to a new version. This requires user interaction as
      * the updated extension will not be installed, until the user grants the new permissions.
+     * @param onInstalledExtensions (optional) indicates that all installed extensions have been
```

Comment:
nit: `onExtensionsInitialized` (optional) callback invoked when all installed extensions are initialized.

(That's really what it is as we will have registered them at this point.)

Reply:
I was bit hesitant to name it in that way, as I'm not sure we are fully Initialized at that point
as there are some function that haven't run, I was afraid of convey they wrong idea.  

- [registerActionHandlersForNewSessions](https://github.com/mozilla-mobile/android-components/blob/42c32815b69267de19719e2284988c069daf3596/components/support/webextensions/src/main/java/mozilla/components/support/webextensions/WebExtensionSupport.kt#L117)
- [runtime.registerWebExtensionDelegate](https://github.com/mozilla-mobile/android-components/blob/42c32815b69267de19719e2284988c069daf3596/components/support/webextensions/src/main/java/mozilla/components/support/webextensions/WebExtensionSupport.kt#L119)

Reply:
Yeah, that's fair. I can't really think of a better name right now though. They are fully initialized in the engine though. Maybe `onExtensionsLoaded` and then document that this means the extensions are loaded by the engine but the UI doesn't have to be initialized yet at this point.

Reply:
Let's do that 👍